.azure-pipelines/buildAndPackage.yml

```diff
@@ -30,10 +30,37 @@ pr: none
 pool:
   vmImage: windows-latest
 
-steps:
-- template: templates/checkout-and-credscan.yml
-- template: templates/install-java.yml
-- template: templates/secure-files.yml
-- template: templates/build-and-coverage.yml
-- template: templates/publish-artefacts.yml
-- template: templates/alert-failure.yml
+stages:
+- stage: Build-Stage
+  jobs:
+  - job: Build
+    steps:
+    - template: templates/build-stage/checkout-and-credscan.yml
+    - template: templates/build-stage/install-java.yml
+    - template: templates/build-stage/secure-files.yml
+    - template: templates/build-stage/build-and-coverage.yml
+    - template: templates/build-stage/publish-artefacts.yml
+    - template: templates/alert-failure.yml
+- stage: Maven-Preview-Stage
+  jobs: 
+  - job: Maven-Preview
+    steps: 
+    - template: templates/maven-preview-stage/secure-files.yml
```

Comment:
this is probably missing a "dowload aterfacts" template

Reply:
I believe the ui release definitions do a few extra things you have to explicitly add in yaml. If the current doesn't download artefacts, how does it publish the assets we built at the previous stage today?

Reply:
You're right, there is an artifact download that occurs prior to the other tasks in that stage. This doesn't appear as a task on the UI which is why I missed it. I'm suspect there may be other steps hidden in the other stages which I may have missed. I'll review those also.